### ISSUE TITLE
adding a nonslime head with a brain to a slime destroys the slimes soul

### ISSUE THREAD

**Issue by ARMOKS** [Goob-Station/Goob-Station#877]
## Description
title

**Reproduction**
remove slime head, remove human head, add human head to slime, slime gets ghosted.

**Comment by a-person5660** [comment]
it might be worth making a mega issue instead of individual ones

**Comment by deltanedas** [comment]
this is because the slimes brain+stomach is in its torso, so if you add a new brain it takes over

**Comment by gluesniffler** [comment]
Yeah the only way to prevent this is by preventing adding heads that arent a slime's. Which leads us to part/organ-species compatibility. Yippee.

**Comment by gluesniffler** [comment]
> it might be worth making a mega issue instead of individual ones

Yes please, I'm not a maintainer in this repo so its harder to manage it when theres a shitload of issues, and I can't close individual ones.

**Comment by SX-7** [comment]
can't repro, assumed fixed
